docs/Installation.md

```diff
@@ -170,45 +164,44 @@ Congratulations! You're now ready to use the Kunstmaan Bundles CMS. Browse to:
 And log in with user ```admin``` and password ```admin```.
 
 *NOTE:* Make sure you change at least the password before putting your site online! You can do this by navigating to
-Settings > Users in the back-end, and changing it there or by running the following :
+Settings > Users in the back-end, and changing it there or by running the following:
 
-    app/console fos:user:change-password admin
+    php app/console fos:user:change-password admin
 
 
 6) Summary
 ----------
 
 You should first check that your system setup matches the minimum requirements, but other than that here's the gist
-of it :
-
-```
-curl -s http://getcomposer.org/installer | php
-php composer.phar create-project kunstmaan/bundles-standard-edition path/to/install
-app/console kuma:generate:bundle
-app/console kuma:generate:default-site --demosite
-app/console doctrine:schema:create
-app/console doctrine:fixtures:load
-app/console kuma:generate:admin-tests
-bower install
-npm install
-grunt build
-app/console assets:install web
-app/console assetic:dump
-```
+of it:
+
+    composer create-project kunstmaan/bundles-standard-edition path/to/install
```

Comment:
this doc mixed the indentation and backtick syntax for code blocks, decided to go with the indentation syntax